### PR TITLE
docs(SRV): add containers troubleshooting MTA-3603

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: serverless/containers/troubleshooting/common-errors.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+meta:
+  title: Common errors encountered while using Serverless Containers
+  description: This page suggests solutions to solve common issues with error messages while using Serverless Containers
+content:
+  h1: Common errors encountered while using Serverless Containers
+  paragraph: This page suggests solutions to solve common issues with error messages while using Serverless Containers
+content:
```
Comment (SamyOubouaziz) [inline serverless/containers/troubleshooting/common-errors.mdx:8]: remove content duplicate

**Review by bene2k1 — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

File: serverless/containers/troubleshooting/common-errors.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Unable to deploy the image
+
+### Possible solutions
+
+- Make sure the container is bound to `0.0.0.0`.
+
+- Make sure the container is listening on port `$PORT` or `8080`.
+
+- Check if KSVC is in `RevisionMissing` status.
```
Comment (bene2k1) [inline serverless/containers/troubleshooting/common-errors.mdx:24]: Mabe explain what KSVC stands for?

File: serverless/containers/troubleshooting/common-errors.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    ```
+    docker inspect myimage
+    ```
+  </Message>
+
+## Unable to build
+
+### Possible solutions
+
+- Make sure that you used a public image from the [Docker Registry](https://docs.docker.com/registry/) or a correct image from the [Scaleway Container Registry](/serverless/containers/how-to/deploy-a-container-from-scaleway-container-registry/).
```
Comment (bene2k1) [inline serverless/containers/troubleshooting/common-errors.mdx:43]:
```suggestion
- Make sure that you used a public image from the [Docker Registry](https://docs.docker.com/registry/) or a compatible image from the [Scaleway Container Registry](/serverless/containers/how-to/deploy-a-container-from-scaleway-container-registry/).
```

File: serverless/functions/troubleshooting/index.mdx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 meta:
-  title: Serverless Functions - Troubleshooting Documentation
+  title: Containers - Troubleshooting
```
Comment (bene2k1) [inline serverless/functions/troubleshooting/index.mdx:3]:
```suggestion
  title: Functions - Troubleshooting
```

File: serverless/functions/troubleshooting/index.mdx [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
   description: Serverless Functions Troubleshooting Documentation
 content:
-  h1: Serverless Functions Troubleshooting Documentation
+  h1: Containers - Troubleshooting
```
Comment (bene2k1) [inline serverless/functions/troubleshooting/index.mdx:6]:
```suggestion
  h1: Functions - Troubleshooting
```

**Pull request merged**